closed/src/java.base/share/classes/com/sun/crypto/provider/NativeCipherBlockChaining.java

```diff
@@ -182,7 +182,10 @@ void init(boolean decrypting, String algorithm, byte[] key, byte[] iv)
 
         mode = (decrypting) ? 0 : 1;
 
-        int ret = nativeCrypto.CBCInit(nativeContext, mode, iv, iv.length, key, key.length);
+        int ret = 0;
```

Comment:
Please remove the unnecessary initialization here, and similar places throughout this file.

Reply:
Removed.